composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "drupal/crop": "^2.1",
-        "dpc-sdp/tide_core": "^3.0.1",
+        "dpc-sdp/tide_core": "dev-feature/SRM-512_update_drupal_to_9.4",
```

Comment:
Revert it back after tide_core release and before merging this PR.

Reply:
Changes reverted